### DISCUSSION TITLE
Differences in image scanning results between DockerHub and Trivy.

### DISCUSSION THREAD

**Discussion by EmilTK** [aquasecurity/trivy#3920] [Q&A] [answered]
Hello! From February 27th, 2023, Docker began rolling out changes to the technology that supports the Docker Hub Vulnerability Scanning feature. The scan results Trivy and DockerHub are different, for example image oraclelinux:8.7, trivy result is 1 high vulnerability, DockerHub result is 27 high vulnerabilities, why this difference?

**Comment by itaysk** [comment, marked as answer]
It's two different scanning engines, different vulnerability databases, you should expect discrepancies. If you think Dockerhub identified something correctly where Trivy did not, please open an issue with the specific CVE and test case and we will look into the case.
